system/ee/ExpressionEngine/Controller/Settings/SecurityPrivacy.php

```diff
@@ -163,7 +163,15 @@ public function index()
                     'title' => 'require_secure_passwords',
                     'desc' => 'require_secure_passwords_desc',
                     'fields' => array(
-                        'require_secure_passwords' => array('type' => 'yes_no')
+                        'require_secure_passwords' => array(
+                            'type' => 'radio',
+                            'choices' => array(
+                                'n' => lang('password_security_none'),
```

Comment:
For readability I think we should change these from letters to words.

Reply:
Tagging @intoeetive

Reply:
We have labels for the readability. The options are letters for historical reasons - in previous versions the possible values for this setting were `y` and `n`, so we're adding two more here following the same pattern